app/templates/components/events/view/overview/event-tickets.hbs

```diff
@@ -19,25 +19,29 @@
       </tr>
     </tbody>
   </table>
-  <h4 class="ui header">{{t 'Details'}}</h4>
-  <table class="ui very basic compact table">
-    <thead>
-      <tr>
-        <th>{{t 'Type'}}</th>
-        <th>{{t 'Sold'}}</th>
-        <th>{{t 'Total'}}</th>
-        <th>{{t 'Available'}}</th>
-      </tr>
-    </thead>
-    <tbody>
-      {{#each data.tickets as |ticket|}}
+  {{#if (not-eq data.tickets.length 0)}}
+    <h4 class="ui header">{{t 'Details'}}</h4>
+    <table class="ui very basic compact table">
+      <thead>
         <tr>
-          <td>{{ticket.name}}</td>
-          <td>{{ticket.orderStatistics.tickets.completed}}</td>
-          <td>{{ticket.quantity}}</td>
-          <td>{{sub ticket.quantity ticket.orderStatistics.tickets.completed}}</td>
+          <th>{{t 'Type'}}</th>
+          <th>{{t 'Sold'}}</th>
+          <th>{{t 'Total'}}</th>
+          <th>{{t 'Available'}}</th>
         </tr>
-      {{/each}}
-    </tbody>
-  </table>
+      </thead>
+      <tbody>
+        {{#each data.tickets as |ticket|}}
+          <tr>
+            <td>{{ticket.name}}</td>
+            <td>{{ticket.orderStatistics.tickets.completed}}</td>
+            <td>{{ticket.quantity}}</td>
+            <td>{{sub ticket.quantity ticket.orderStatistics.tickets.completed}}</td>
+          </tr>
+        {{/each}}
```

Comment:
Use the each else syntax of ember instead of adding another if statement 
https://guides.emberjs.com/release/templates/displaying-a-list-of-items/

Check how it's done for the manage roles table.